docs/docs/deploying-to-digitalocean-droplet.md

```diff
@@ -0,0 +1,247 @@
+
+DigitalOcean provides a cloud platform to deploy, manage, and scale applications of any size, removing infrastructure friction and providing predictability so developers and their teams can spend more time building better software.
+
+DigitalOcean's product droplets are scalable compute IaaS (Infrastructure as a Service) or a VPS (Virtual Private Server) on the cloud which has great reliability and scalability. They come with varied price ranges ideal for small apps to giant enterprise-level apps.
+
+They provide service to select from various Unix-based distributions and select your technology-based platform with preinstalled pre-requisites from the marketplace. This guide will walk through the specific options that work best for deploying a Gatsby site with DigitalOcean.
+
+A Droplet can be spun up in less than a minute for as little as \$5/month.
+
+## Prerequisites
+
+- A Gatsby site living in a Git repository (GitHub, GitLab, or any Git cloud)
+- A [DigitalOcean Droplet](https://www.digitalocean.com/products/droplets/) with a non-root user configured with sudo group ([example: Ubuntu 18.04](https://www.digitalocean.com/community/tutorials/how-to-create-a-new-sudo-enabled-user-on-ubuntu-18-04-quickstart))
+- A custom domain name for your Gatsby site to help with configuring HTTPS
+
+## How to deploy your Gatsby site to DigitalOcean
+
+### Install NodeJS, NPM and Gatsby-CLI onto your droplet
+
+Follow these instructions for installs on an Ubuntu droplet.
+
+1. Log in to your droplet as a non-root user.
+
+2. Install NodeJS
+
+   ```bash
+   sudo apt-get update
+   sudo apt-get install nodejs
+   ```
+
+3. Install npm
+
+   ```bash
+   sudo apt-get install npm
+   ```
+
+   To view the version of Node and npm installed, run,
+
+   ```bash
+   node -v
+   npm -v
+   ```
+
+4. To install the latest stable Node.js release using the `n` package (Required),
+
+   ```bash
+   sudo npm install -g n
+   sudo n stable
+   ```
+
+   `Note`: If you check the version now, you would see the older versions of node and npm from the cache.
+   You can either exit and restart your terminal or refresh the cache by following commands,
+
+   ```bash
+   hash node
+   hash npm
+   ```
+
+5. Install the gatsby-cli now globally. This will be useful ahead in building the Gatsby site for production.
+
+   ```bash
+   sudo npm install -g gatsby-cli
+   ```
+
+### Clone your repository to the droplet
+
+The next step is to clone the repository containing your Gatsby app (Replace `<your-github-repo-site>` with your Github repository link)
+
+```bash
+git clone <your-github-repo-site>
+```
+
+Incase of any warning related to Permission denied, check if <your non-root user> have sudo privileges or before cloning your repository, give permission to <your non-root user> to access the .config directory of under /home/gatsby/.
```

Comment:
```suggestion
In case of a warning related to "Permission denied", check if `<your non-root user>` has `sudo` privileges. Or before cloning your repository, [change permissions](https://help.ubuntu.com/community/FilePermissions) for `<your non-root user>` to access the `.config` directory of under `/home/gatsby/':
```